automation/test.sh

```diff
@@ -27,6 +27,18 @@ readonly TEMPLATES_SERVER="gs://kubevirt-vm-images"
 readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
 
 
+# Skip if it's docs changes only
+if [ ${CI} == "true" ]; then
+    CI_GIT_ALL_CHANGES=$(git diff --name-only $PULL_BASE_SHA $PULL_PULL_SHA)
+    CI_GIT_SKIP_CHANGES=$(cat <<<$CI_GIT_ALL_CHANGES | grep -E "^(docs/|.*/OWNERS|.*\.md)" || :)
```

Comment:
Some more to consider skipping please png, svg, rst, txt, LICENSE, NOTICE, SECURITY_CONTACTS, PATENTS, MAINTAINERS
(we have some on non docs folder)

nit: consider please adding $ to .md, we dont have a file with .md in the middle, so we should be save anyhow
unless we can expect something weird like that

Reply:
+1 on txt
Generally I'd like to limit the expression to files whicih we expect to changes
Owners, docs.
License, Notice, Security_contacts ... maybe the change some day, but not regularly.

Thus: I'm keeping the list smaller in favor of readbaility.

Technically however you are correct.